client/state/invites/reducer.js

```diff
@@ -83,10 +86,30 @@ export const items = createReducer(
 				[ action.siteId ]: siteInvites,
 			};
 		},
+		[ INVITES_DELETE_REQUEST_SUCCESS ]: ( state, action ) => {
+			return {
+				...state,
+				[ action.siteId ]: {
+					accepted: deleteInvites( state[ action.siteId ].accepted, action.inviteIds ),
+					pending: deleteInvites( state[ action.siteId ].pending, action.inviteIds ),
+				},
+			};
+		},
```

Comment:
I am seeing some strange behavior here:  invites are deleted, and disappear from the details list immediately (as expected), but then they re-appear later.  It appears that the next API call to list invites (which occurs "immediately" after deletion succeeds) can occasionally return some deleted invites.  Replication delay?  Swamp gas from a weather balloon trapped in a thermal pocket?  I'm not sure, but we will probably want to solve that before merging.

Reply:
Definitely the swamp gas.